README.md

```diff
@@ -49,7 +49,8 @@ Pre-built pip packages support Ubuntu 18.04+, macOS 10.15+ and Windows 10+
 
 ```bash
 # Install
-pip install open3d
+pip install open3d       # or
+pip install open3d_cpu   # Smaller CPU only wheel on Linux (v0.17+)
```

Comment:
Shall we use `open3d-cpu` instead of `open3d_cpu`? Or, do we have some other considerations?

By looking at the [top PyPI packages](https://hugovk.github.io/top-pypi-packages/), seems like the pattern is that `dash-name` is more often used compared with `underscore_name`. For TensorFlow, it is also named `tensorflow-cpu`.

Reply:
Changed instructions to `open3d-cpu`. pip treats `_` and `-` as the same, though. So for example:
```sh
$ pip show open3d_cpu
Name: open3d-cpu
Version: 0.16.1+e9497ae43
Summary: Open3D: A Modern Library for 3D Data Processing.
Home-page: http://www.open3d.org
Author: Open3D Team
Author-email: info@open3d.org
License: MIT
Location: /home/ssheorey/.pyenv/versions/3.8.8/envs/open3d/lib/python3.8/site-packages
Requires: configargparse, dash, nbformat, numpy
Required-by: 
```
One requirement is that the wheel file name must have '_', *not* '-' otherwise the version parsing is incorrect.